### PR TITLE
Doglist enhanced 

### PR DESCRIPTION
**Pull request by MariaSLapshina**
hi review pls

### CODE REVIEW

File: client/Components/Dogs/Dogs.jsx [hunk 1]
```diff
@@ -25,14 +26,29 @@ class Dogs extends Component {
...
+      <div id ='dog' key={dog.id} >
+      <h1><Link to={`${DOGS}/${dog.id}`}>{dog.name}</Link></h1>
+        <div id='dogblock'>
+          <div id ='dogtext'>
+          Price: ${dog.price}<br/>
+          Gender: {dog.gender}<br/>
+          Age: {dog.age} {dog.ageUnit}s<br/>
+          Size: {dog.size}<br/>
+          Breed: {dog.breed.name}
+            <p id ='description'>{dog.description}</p>
```
Comment (clacayetano) [inline client/Components/Dogs/Dogs.jsx]: This is more just a question. So the only difference between dog list and dog view right now is the `description` right?

Comment (MariaSLapshina) [reply client/Components/Dogs/Dogs.jsx]: that is a good question. i didnt know which info is appropriate to show on dog list, i will chop off any extras u guys think shouldnt be there

Comment (MariaSLapshina) [reply client/Components/Dogs/Dogs.jsx]: actually they both have pretty much the same info so i can def remove some of it from list

File: client/Components/Dogs/dogs.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+@import '../../variables.scss';
+#dogs {
+    margin:10% auto;
+    display: flex;
```
Comment (clacayetano) [inline client/Components/Dogs/dogs.scss:9]: we may want to add `flex-display: wrap`. right now with more than 3 dogs, it looks like this

File: client/Components/Dogs/Dogs.jsx (hunk 1 above)
Comment (clacayetano) [reply client/Components/Dogs/Dogs.jsx]: i'm good with this one... just mentioning that currently that's the only difference. right now having all the info there (except description makes it easier to test filter)

**Comment by clacayetano** [comment]
This looks good! but merging in the latest from master might ruin the current Dog List design (because of the FilterSideBar, which not yet styled)

File: client/Components/Dogs/Dogs.jsx (hunk 1 above)
Comment (MariaSLapshina) [reply client/Components/Dogs/Dogs.jsx]: they both have the description..if u go to dog view look on the card itself, right below the picture - there is  a description

File: client/Components/Dogs/dogs.scss (hunk 2 above)
Comment (MariaSLapshina) [reply client/Components/Dogs/dogs.scss:9]: truee

**Comment by clacayetano** [comment]
CodeClimate is mad because Dogs's `render` block is too large. Would you rather we disable that check? Or are you okay with refactoring the Dog block into its own component?

File: client/Components/Dogs/Dogs.jsx (hunk 1 above)
Comment (inKEYYgnito) [reply client/Components/Dogs/Dogs.jsx]: maybe we just add photo, name, breed, and add to card per dog on dog list view.

Comment (MariaSLapshina) [reply client/Components/Dogs/Dogs.jsx]: Ok. I will refactor the dog block accordingly so codeclimate also isnt mad

File: client/Components/Dogs/Dogs.jsx [hunk 3]
```diff
@@ -50,19 +51,32 @@ class Dogs extends Component {
...
+            size={size}
+          />
+        </div>
+        <div id='dogs'>
+          {dogsToDisplay.map(dog => (
+            <div id='dog' key={dog.id} >
+              <h1><Link to={`${DOGS}/${dog.id}`}>{dog.name}</Link></h1>
+              <div className='dogcontent'>
+                <div id='dogtext'>
+                  Breed: {dog.breed.name}
```
Comment (inKEYYgnito) [inline client/Components/Dogs/Dogs.jsx]: Take off 'Breed:' and just leave breed value

**Review by inKEYYgnito — CHANGES_REQUESTED** [review]

File: client/Components/Dogs/dogs.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    border: solid 3px $intense-pink;
+    border-style: outset;
+}
+.filter{
+    flex-basis: 20%;
+}
+
+button.btn-add-crate {
+    --btn-crate-font-size: 1em;
+  }
```
Comment (inKEYYgnito) [inline client/Components/Dogs/dogs.scss:51]: indent pls

**Review by inKEYYgnito — APPROVED** [review]

File: client/Components/Dogs/dogs.scss (hunk 4 above)
Comment (MariaSLapshina) [reply client/Components/Dogs/dogs.scss:51]: NO

**Pull request merged**